wagtail/admin/ui/side_panels.py

```diff
@@ -337,81 +301,12 @@ class SidePanelToggle(BaseSidePanel.SidePanelToggle):
     template_name = "wagtailadmin/shared/side_panels/preview.html"
     order = 400
 
-    def get_context_data(self, parent_context):
-        context = super().get_context_data(parent_context)
-        context["has_multiple_modes"] = len(self.object.preview_modes) > 1
-        return context
-
+    def __init__(self, object, request, *, preview_url):
+        super().__init__(object, request)
+        self.preview_url = preview_url
 
-class PagePreviewSidePanel(BasePreviewSidePanel):
     def get_context_data(self, parent_context):
         context = super().get_context_data(parent_context)
-        if self.object.id:
-            context["preview_url"] = reverse(
-                "wagtailadmin_pages:preview_on_edit", args=[self.object.id]
-            )
-        else:
-            content_type = parent_context["content_type"]
-            parent_page = parent_context["parent_page"]
-            context["preview_url"] = reverse(
-                "wagtailadmin_pages:preview_on_add",
-                args=[content_type.app_label, content_type.model, parent_page.id],
-            )
+        context["preview_url"] = self.preview_url
+        context["has_multiple_modes"] = len(self.object.preview_modes) > 1
         return context
-
-
-class BaseSidePanels:
-    def __init__(self, request, object):
-        self.request = request
-        self.object = object
-
-        self.side_panels = [
-            BaseStatusSidePanel(object, self.request),
-        ]
-
-    def __iter__(self):
-        return iter(sorted(self.side_panels, key=lambda p: p.order))
-
-    @cached_property
-    def media(self):
-        media = Media()
-        for panel in self:
-            media += panel.media
-        return media
-
-
-class PageSidePanels(BaseSidePanels):
```

Comment:
@laymonage we need upgrade considerations for this.

Just spent a good 15 min finding the PR with the change (and that is with me vaguely remembering seeing the `PageSidePanels` removal. We use this on wagtail-localize